apps/partner/client/articles.coffee

```diff
@@ -1,12 +1,57 @@
+_ = require 'underscore'
 ArticlesGridView = require '../../../components/articles_grid/view.coffee'
+ArticleView = require '../../../components/article/client/view.coffee'
 Articles = require '../../../collections/articles.coffee'
+Article = require '../../../models/article.coffee'
+articleTemplate = -> require('../../../components/article/templates/index.jade') arguments...
 sd = require('sharify').data
+{ resize } = require '../../../components/resizer/index.coffee'
+embed = require 'particle'
+moment = require 'moment'
 
 module.exports = class ArticlesAdapter
-  constructor: ({ profile, partner, cache, el }) ->
-    collection = new Articles
-    collection.url = "#{collection.url}/?partner_id=#{partner.get('_id')}&published=true&limit=12&sort=-published_at"
-    view = new ArticlesGridView el: el, collection: collection
-    el.html '<div class="loading-spinner"></div>'
-    collection.fetch()
-    view
\ No newline at end of file
+  constructor: ({ @profile, @partner, @cache, @el }) ->
+    @renderArticle() if @isArticle()
+    @renderArticlesGrid() if not @isArticle()
+
+  isArticle: ->
+    window.location.pathname.includes '/article/'
```

Comment:
Its there a stronger way to evaluate this? Like checking for the presence of `sd.ARTICLE`?